pkg/api/registry/registry.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.14.0
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.6.1
```

Comment:
I'm not sure that we should change these versions.

@NikitaSkrynnik Could you re-generate it with 
```
// 	protoc-gen-go v1.26.0
// 	protoc        v3.14.0
```?